packages/amplify-e2e-core/src/utils/index.ts

```diff
@@ -30,7 +30,7 @@ export * from './git-operations';
 /**
  * Whether the current environment is CircleCI or not
  */
-export const isCI = (): boolean => process.env.CI && process.env.CIRCLECI;
+export const isCI = (): boolean => JSON.parse(process.env.CI && process.env.CIRCLECI);
```

Comment:
should it be `JSON.parse(process.env.CI) && JSON.parse(process.env.CIRCLECI)`?;

Reply:
![image](https://user-images.githubusercontent.com/85319655/201712108-f995ab6f-b1b0-4af1-b7a5-2de4a716cf53.png)

Reply:
> Should we add it back? or try Boolean(process.env.CI && process.env.CIRCLECI ?

`process.env` will always return a string, so it would return `{}`. I think the assumption is that both `process.env.CI` and `process.env.CIRCLECI` return stringified booleans from CI. `Boolean()` would not return what we want though.

``js
Boolean('false') // => true
JSON.parse('false') // => false
``

What I do need to do is change this to `JSON.parse(process.env.CI) && JSON.parse(process.env.CIRCLECI)` because `string && string` returns the latter value.

Reply:
```
Boolean('false') // => true
JSON.parse('false') // => false
```
![image](https://user-images.githubusercontent.com/5849952/201721139-a03f97ef-2a75-40e7-85e4-f0d33234b5bc.png)